Sources/Snowplow/Tracker/LogLevel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  LogLevel.swift
+//  Snowplow
+//
+//  Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+//
+//  This program is licensed to you under the Apache License Version 2.0,
+//  and you may not use this file except in compliance with the Apache License
+//  Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+//  http://www.apache.org/licenses/LICENSE-2.0.
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the Apache License Version 2.0 is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+//  express or implied. See the Apache License Version 2.0 for the specific
+//  language governing permissions and limitations there under.
+//
+//  Authors: Alex Benini
+//  License: Apache License Version 2.0
+//
+
+import Foundation
+
+@objc(SPLogLevel)
+public enum LogLevel : Int {
+    case off = 0
```

Comment:
question: why does `off` have a number but the others don't?

Reply:
It's enough for the first value in the enum to have an explicit value, the others have an implicit value – incremented by 1 for each one in the sequence.